marklogic-unit-test-client/src/test/ml-modules/root/test/suites/Helpers/Delete All XML/test-data/td.xqy

```diff
@@ -0,0 +1,10 @@
+xquery version "1.0-ml";
```

Comment:
This file looks like it should live in a /lib directory, not inside test-data.

Reply:
I was following the guide over at https://github.com/marklogic-community/roxy/wiki/Unit-Testing which suggested housing common stuff that setups/teardowns use. If we put it in a /lib would it have to go outside the test suite dir?

Reply:
Yeah, we should probably update that documentation.  While it hasn't been merged onto master yet, develop now supports `lib/` directories:

https://github.com/marklogic-community/marklogic-unit-test/blob/6e3304a3c1df4c6d7b959fddc730e7bafbaa21d2/marklogic-unit-test-modules/src/main/ml-modules/root/test/test-controller.xqy#L25-L27

Reply:
Oh perfect, I'll update and make some of the other changes you suggested.

Reply:
Thanks @cskeefer.  I'll resolve these conversations when the commits come through.